plugins/jobs/server/__init__.py

```diff
@@ -42,7 +46,18 @@ def scheduleLocal(event):
         fn = getattr(module, job.get('function', 'run'))
         fn(job)
 
+@access.token(scope=constants.REST_CREATE_JOB_TOKEN_SCOPE)
+def _authorizeRestJobCreation(event):
+    tokenModel = ModelImporter.model('token')
+    token = getCurrentToken()
+
+    if not tokenModel.hasScope(token, constants.REST_CREATE_JOB_TOKEN_SCOPE):
+        raise AccessException(
+            'Invalid token scope.\n'
+            'Required: %s.\n' % (constants.REST_CREATE_JOB_TOKEN_SCOPE))
+
 
 def load(info):
     info['apiRoot'].job = job_rest.Job()
+    events.bind('rest.post.job.before', 'jobs', _authorizeRestJobCreation)
```

Comment:
Since this is all in the same plugin, let's just move this logic into the REST route handler itself. That will improve readability via lexical proximity of the logic, and won't preclude behavioral modification by downstreams.